pylabnet/gui/widgets/shutterwidget.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+import ipywidgets as widgets
+
+"""
+This file contains wrappers for Jupyter Notebook Widgets used for controling the SC20 Shutter
+
+"""
+
+
+class Shuttertoogle():
```

Comment:
@cknaut Looks good! A few slight style comments.

- The good style in Python is to use CamelCase naming convention for class names. For example: ```ShutterToggle``` instead of ```Shuttertoggle```

- In the doc string, don't make the first newline. The standard format is:
``` 
""" First line - one-sentence description
    Empty line
    Subsequent lines - full description including arguments.
""" finial quotes on a new line
empty line
Code begins  
```

Reply:
Thanks for the styling suggestions, will change is accordingly!  As a general idea on how to manage style conventions, we could use a linter which automatically checks for styling conventions. Pylint or flake8 are examples of this. We can have a shared styling settings which every contributer has to use. Happy to help setting this up!

Reply:
I think Pycharm had pylint, we can stick to it. This is short reference which I was using https://realpython.com/python-pep8/

Reply:
OK, good idea. There's also a way to let github test code style before every merge: https://github.com/marketplace/hound/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW4xMDYx#pricing-and-setup

Never tried it, but it could be interesting to test.